src/services/message.service.js

```diff
@@ -0,0 +1,15 @@
+const { Message } = require('../models/message.model');
+const { User } = require('../models/user.model');
+
+const getAllByRoom = async (roomId) => {
+  return Message.findAll({
+    where: { roomId },
+    include: [{ model: User, required: true }],
+  });
+};
+
+const create = async (message) => {
+  return Message.create(message);
```

Comment:
Consider adding validation for the `message` object to ensure it meets the expected format and constraints before attempting to create a new message.